docs/general/rendering.md

```diff
@@ -243,6 +243,28 @@ This will be rendered as:
 ```
 Note: the `Attributes` adapter (default) does not include a resource root. You also will not be able to create a single top-level root if you are using the :json_api adapter.
 
+#### namespace
+
+The namespace for serializer lookup is based on the controller, via `ActionController::Serialization#namespace_for_serializer`.
+
+To configure the implicit namespace, in your controller, define
+
```

Comment:
i.e. below is the current implementation.  ooh,, this would be a good reason to add a setter

```
attr_writer :namespace_for_serializer
```

makes it cleaner and can do in a before filter ?